### DISCUSSION TITLE
Add links to Symbiota Docs documentation from tools in Symbiota portals

### DISCUSSION THREAD

**Discussion by themerekat** [Symbiota/community-board#27] [Wish List]
What if we used the green question marks next to the tools in the control panels to indicate where to learn more about these tools? See highlighting in this screenshot:

![image](https://user-images.githubusercontent.com/16688784/141539339-ea9e930d-7e0b-4534-80f0-46836bf00c1b.png)

**Comment by arbolitoloco** [comment]
I have been developing documentation for all pages in Symbiota and the question marks directly in Symbiota code using an external documentation provider called [symbiota-tooltips](https://github.com/biokic/symbiota-tooltips), so this will be done eventually! I will need help with adding the text to the pages and terms in the tooltips repository FYI!

**Reply by themerekat** [reply]
Excellent! Let me know when I can help.
